### DISCUSSION TITLE
Tuần 5

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#36] [General]
Đọc project tuần này để biết phải làm j nka, việc nhẹ hoi. Làm xong rồi up file lên chỗ này, mn thấy oke thì up lên code tuần 6 mình bắt đầu trang trí nhe.

**Comment by TranBaoLam10C13** [comment]
Bài tui xịn r mà

**Reply by dangxuantruong10a13** [reply]
Chứ j nữa, chỉ là tui tạo việc làm thoi ;))

**Reply by TranBaoLam10C13** [reply]
:))

**Comment by 04lananh** [comment]
Ủa vậy là tui chỉ cần kiếm hình gòi chỉnh chữ hoi đúng ko 
nd có cần chỉnh j nx ko

**Reply by dangxuantruong10a13** [reply]
Yes, thêm nhiều kĩ thuật thầy đưa nhất nếu thấy bài mình có thể thêm đc kkk

**Comment by NgHan12pn** [comment]
Z cuối cùng là up file ở đây hay up lên code lun

**Reply by dangxuantruong10a13** [reply]
Là sửa xong bỏ lên đây xem thêm lần nữa r lưu vô code kkkkk

**Reply by NgHan12pn** [reply]
Okayy thks @dangxuantruong10a13

**Comment by dangxuantruong10a13** [comment]
[bản thân tôi.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8311605/b.n.than.toi.docx)
[kết bài kkkkk.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8311606/k.t.bai.kkkkk.docx)
hú le tui để bài ở đây nha kkkk mn vào coi

**Comment by dangxuantruong10a13** [comment]
Lan Anh quên nói là bà quên viết phê phán, làm sao, các kiểu á viết đi nha oh sjt

**Reply by 04lananh** [reply]
ủa viết gòi mà đại ka

**Reply by dangxuantruong10a13** [reply]
áaaaaaaaaa

**Reply by dangxuantruong10a13** [reply]
còn cái cách để sống tử tế cần thêm vô hong
